### PR TITLE
chore: bump cli versions

### PR DESCRIPTION
**Pull request by sobolk**
#### Description of changes

This PR absorbs latest CLI beta

#### Description of how you validated changes

E2E test run https://app.circleci.com/pipelines/github/aws-amplify/amplify-category-api/4122/workflows/1e70647f-108b-48ad-8c58-73affc81f503 

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] Any CDK or CloudFormation parameter changes are called out explicitly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-e2e-core/src/init/amplifyPush.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ export function cancelIterativeAmplifyPush(
       .wait(/.*AWS::AppSync::GraphQLSchema.*UPDATE_IN_PROGRESS.*/)
       .sendCtrlC()
       .run((err: Error) => {
-        if (err && !/Killed the process as no output receive for/.test(err.message)) {
+        if (err && !/Process exited with non zero exit code 130/.test(err.message)) {
```
Comment (sobolk) [inline packages/amplify-e2e-core/src/init/amplifyPush.ts:145]: related to inquirer -> prompter migration in cli.

File: packages/amplify-migration-tests/src/__tests__/migration_tests_v10/__snapshots__/api-rest-lambda.migration.test.ts.snap [hunk 2]
```diff
@@ -3,6 +3,56 @@
...
+      .promise()
+      .then((res) => {
+        response.send(event, context, response.SUCCESS, { appSecret: res.UserPoolClient.ClientSecret });
+      })
+      .catch((err) => {
+        response.send(event, context, response.FAILED, { err });
+      });
+  }
+};
+
```
Comment (sobolk) [inline packages/amplify-migration-tests/src/__tests__/migration_tests_v10/__snapshots__/api-rest-lambda.migration.test.ts.snap:55]: Drifts in this file are expected. Therefore recording them.
They come from re-formatted lambdas in auth category.
See https://github.com/aws-amplify/amplify-cli/pull/12028#discussion_r1110030827 for reference.

File: package.json [hunk 3]
```diff
@@ -139,14 +139,14 @@
...
-    "**/amplify-cli-logger": "1.2.4-beta.0",
-    "**/amplify-cli-shared-interfaces": "1.1.1-beta.3",
-    "**/amplify-function-plugin-interface": "1.9.6-beta.3",
-    "**/amplify-headless-interface": "1.17.1-beta.0",
-    "**/amplify-storage-simulator": "1.7.3-beta.0",
-    "**/amplify-util-headless-input": "1.9.10-beta.0"
+    "**/amplify-cli-core": "4.0.0-beta.7",
+    "**/@aws-amplify/amplify-environment-parameters": "1.4.0-beta.1",
+    "**/@aws-amplify/amplify-category-function": "5.0.0-beta.1",
+    "**/@aws-amplify/amplify-cli-logger": "1.3.0-beta.0",
```
Comment (sobolk) [inline package.json:145]: we're in process of adding `@aws-amplify` scope to cli packages. Hence changes like this in the PR.

File: package.json [hunk 4]
```diff
@@ -39,7 +39,7 @@
     "update-versions": "lerna version --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version",
     "commit": "git-cz",
     "coverage": "codecov || exit 0",
-    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@^11.0.0-beta.1 -W  && git checkout -- package.json",
+    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@11.0.0-beta.7 -W  && git checkout -- package.json",
```
Comment (sobolk) [inline package.json:42]: We stopped shipping `cdkv2` tag (which `^` would resolve...) therefore pinning version here.

**Review by phani-srikar — APPROVED** [review]

**Review by jhockett — APPROVED** [review]

**Pull request merged**